Features relying on jack_net can not be built using pipewire-jack

Pipewire-jack provides a `libjacknet.so`, but it does not actually implement jack_net (tracked upstream in: https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/2043).

This leads to build failures with a default build (i.e. `meson build && ninja -C build`):

```
[72/101] Linking target example-clients/jack_net_master
FAILED: example-clients/jack_net_master
cc  -o example-clients/jack_net_master example-clients/jack_net_master.p/netmaster.c.o -Wl,--as-needed -Wl,--no-undefined -Wl,--start-group -ljacknet -Wl,--end-group
/usr/bin/ld: example-clients/jack_net_master.p/netmaster.c.o: in function `signal_handler':
/home/dave/git/audio/jack-example-tools/build/../example-clients/netmaster.c:41: undefined reference to `jack_net_master_close'
/usr/bin/ld: example-clients/jack_net_master.p/netmaster.c.o: in function `main':
/home/dave/git/audio/jack-example-tools/build/../example-clients/netmaster.c:113: undefined reference to `jack_net_master_open'
/usr/bin/ld: /home/dave/git/audio/jack-example-tools/build/../example-clients/netmaster.c:179: undefined reference to `jack_net_master_send_slice'
/usr/bin/ld: /home/dave/git/audio/jack-example-tools/build/../example-clients/netmaster.c:186: undefined reference to `jack_net_master_recv_slice'
/usr/bin/ld: /home/dave/git/audio/jack-example-tools/build/../example-clients/netmaster.c:195: undefined reference to `jack_net_master_close'
collect2: error: ld returned 1 exit status
[76/101] Linking target example-clients/jack_net_slave
FAILED: example-clients/jack_net_slave
cc  -o example-clients/jack_net_slave example-clients/jack_net_slave.p/netslave.c.o -Wl,--as-needed -Wl,--no-undefined -Wl,--start-group -ljacknet -Wl,--end-group
/usr/bin/ld: example-clients/jack_net_slave.p/netslave.c.o: in function `signal_handler':
/home/dave/git/audio/jack-example-tools/build/../example-clients/netslave.c:36: undefined reference to `jack_net_slave_close'
/usr/bin/ld: example-clients/jack_net_slave.p/netslave.c.o: in function `main':
/home/dave/git/audio/jack-example-tools/build/../example-clients/netslave.c:128: undefined reference to `jack_net_slave_open'
/usr/bin/ld: /home/dave/git/audio/jack-example-tools/build/../example-clients/netslave.c:135: undefined reference to `jack_set_net_slave_process_callback'
/usr/bin/ld: /home/dave/git/audio/jack-example-tools/build/../example-clients/netslave.c:136: undefined reference to `jack_set_net_slave_shutdown_callback'
/usr/bin/ld: /home/dave/git/audio/jack-example-tools/build/../example-clients/netslave.c:138: undefined reference to `jack_net_slave_activate'
collect2: error: ld returned 1 exit status
```

Currently it is only possible to build this project without the tooling relying on jacknet (i.e. `meson -Djack_net=disabled build && ninja -C build`).

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -50,8 +50,7 @@ jobs:
         run: pacman --noconfirm -Syu alsa-lib base-devel celt meson opus readline libsamplerate libsndfile zita-alsa-pcmi zita-resampler
       - name: Install pipewire-jack
         run: pacman --noconfirm -S pipewire-jack
-      # disable jack_net support for now: https://github.com/jackaudio/jack-example-tools/issues/61
       - name: Build jack-example-tools
-        run: meson -Djack_net=disabled build && ninja -C build
+        run: meson build && ninja -C build
       - name: Install jack-example-tools
         run: ninja -C build install

--- a/README.md
+++ b/README.md
@@ -44,10 +44,6 @@ options](https://mesonbuild.com/Builtin-options.html#universal-options) (e.g.
 meson --prefix=/usr build
 ```
 
-**NOTE**: Currently it is not possible to use pipewire-jack to build tooling
-that requires **jack_net** support (see
-[#61](https://github.com/jackaudio/jack-example-tools/issues/61)).
-
 To build the applications and libraries [ninja](https://ninja-build.org/) is
 required:
 

